soh/soh/Enhancements/gameplaystats.h

```diff
@@ -21,7 +21,8 @@ typedef enum {
     /* 0xA7 */ TIMESTAMP_DEFEAT_TWINROVA,       // z_boss_tw.c
     /* 0xA8 */ TIMESTAMP_DEFEAT_GANONDORF,      // z_boss_ganon.c
     /* 0xA9 */ TIMESTAMP_DEFEAT_GANON,          // z_boss_ganon2.c
-    /* 0xAA */ TIMESTAMP_MAX
+    /* 0xAA */ TIMESTAMP_FOUND_GREG,            // z_parameter.c
```

Comment:
the reason i added this here instead of using the itemid from z64item.h is that greg is a rando-exclusive item that doesn't have an entry in z64item.h (it would just be green rupee, in which case we'd just have a timestamp the first time link picked up a green rupee off the ground)